.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+# Description of PR purpose/changes
+
+* Please include a summary of the change and which issue is fixed. 
+* Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+## Jira Ticket / Issue #
+e.g. https://kbase-jira.atlassian.net/browse/DATAUP-X
+
+## Type of change
+
```

Comment:
Is the type of change section needed if we are linking to the jira ticket?

Reply:
removed

Reply:
I guess I have a broader question that follows on from this question: Is the PR meant to be self contained, or is it expected the reader has access to (possibly private) external resources?

I'm imagining a developer in the future looking at some PR trying to diagnose a bug, and either we've stopped using JIRA and our account is gone or the developer can't access the ticket for some other reason (e.g. maybe a 3rd party dev).

Reply:
@MrCreosote It is possible to have multiple types of contributing docs for each repo. I think this is for internal developers only right now. 
Yea it's not open source friendly to link to private tickets in Jira, but I don't think it's a priority, I think this is mostly for internal devs to be able to reference back to the board quickly